ATL/schema.sql

```diff
@@ -0,0 +1,69 @@
+Create Table Passengers(
+Id int identity(1,1) Primary key,
+[Name] varchar(30),
+[Surname] varchar(50),
+Age int Not null,
+Check(Age>0)
+)
+
+ON
+
+Create Table AirLines(
+Id int identity(1,1) Primary key,
+[Name] varchar(50) not null,
+Concourse char not null
+)
+
+ON
+
+Create Table Flights(
+Id int identity(1,1) Primary key,
+FlightNumber int not null,

```

Comment:
```
The flight number. For example, “900”. Just know that we sometimes re-use flight numbers.
```
Since flight number can be reused, we can't use it to uniquely identify the flight. Plus if we use flight_id here we can add a Foreign Key relationship and make queries faster and improve data integrity.